src/ImageProcessorCore/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -44,6 +46,7 @@ private class errEOF : Exception { }
         // but in practice, their use is described at
         // http://www.sno.phy.queensu.ca/~phil/exiftool/TagNames/JPEG.html
         private const int app0Marker = 0xe0;
```

Comment:
I'll be moving all these eventually to a constants file eventually when I get this cleaned up.

See the now defunct jpeg branch for an example.

https://github.com/JimBobSquarePants/ImageProcessor/blob/e85458fa7f56903f023d2aa3d70bf2ea0b0f63e8/src/ImageProcessorCore/Formats/Jpg/New/JpegConstants.cs